.editorconfig

```diff
@@ -33,3 +33,6 @@ dotnet_diagnostic.SA1601.severity = suggestion
 
 # Allow FIXME tag as position free
 dotnet_diagnostic.S1134.severity = none
+
+[*.{csproj,xml}]
+indent_size = 2
```

Comment:
Though it configures `indent_size` as `2` but it seems there are `csproj`s which don't follow the rule. It'll be better to add a new GH Actions workflow to check editorconfig.

Reply:
@moreal That's cool!
https://github.com/planetarium/lib9c/issues/1674